wallet-next/src/storage.rs

```diff
@@ -19,9 +23,9 @@ impl Storage {
         // Insert the task, then obtain the ID of this row
         let id = sqlx::query!(
             r#"
-INSERT INTO penumbra ( value )
-VALUES ( ?1 )
-        "#,
+            INSERT INTO penumbra ( value )
+            VALUES ( ?1 )
+            "#,
```

Comment:
This whole method can be dropped, since we don't need it for testing the sqlx queries any more.